src/DynamoCoreWpf/Properties/Resources.resx

```diff
@@ -552,10 +552,6 @@
     <value>Number Format</value>
     <comment>Setting menu | Number format</comment>
   </data>
-  <data name="DynamoViewSettingMenuRenderPrecision" xml:space="preserve">
```

Comment:
This is duplicate of `PreferencesViewVisualSettingsRenderPrecision`. @RobertGlobant20 @Astul-Betizagasti when you have time, let's review the resource strings we touched and make sure no duplicates were left.